Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:2.5
+RUN apt-get update -qq
+RUN mkdir /myapp
+WORKDIR /myapp
```

Comment:
I have a couple of minor recommendations for authoring Dockerfile:
- `WORKDIR` creates a directory in case it does not exist, so usually you may remove additional RUN with mkdir to a have one layer less - but there's a caveat that sometimes WORKDIR does not respect the actual USER and fails with permissions denied, so check it out whether it is working.
- `COPY` instruction can copy multiple files at the same time, so you could simply go for 
```docker file
  COPY Gemfile Gemfile.lock ./
```
- I cannot see any usage of apt-get except for the update statement, so I believe you can remove `RUN apt-get update` until you need to install some tool like certificates manager or something similar.

I would also advice you to check out the multi-stage builds in Docker either through reference or look at https://www.georg-ledermann.de/blog/2018/04/19/dockerize-rails-the-lean-way/ - there are some useful tips on how to reduce the build size and overall approach the building Dockerfile  for your Rails app.

Reply:
I don't write this Dockerfile, it is from docs.docker.com - rails quickstart guide

But thanks for remark. I will edit this lines.